cmd/crio/config.go

```diff
@@ -91,7 +91,7 @@ stream_tls_ca = "{{ .StreamTLSCA }}"
 
 # default_runtime is the _name_ of the OCI runtime to be used as the default.
 # The name is matched against the runtimes map below.
-default_runtime = {{ .DefaultRuntime }}
+default_runtime = "{{ .DefaultRuntime }}"
```

Comment:
Please include the fix directly in the commit introducing the change.

Reply:
I'll squash them before merging, no worry :)